ui/dashboards/src/components/DashboardToolbar.tsx

```diff
@@ -36,47 +38,69 @@ export const DashboardToolbar = (props: DashboardToolbarProps) => {
   };
 
   return (
-    <Toolbar disableGutters sx={{ display: 'block', padding: (theme) => theme.spacing(2, 0) }}>
+    <Toolbar
+      disableGutters
+      sx={{
+        display: 'block',
+      }}
+    >
       {isEditMode ? (
-        <>
+        <Stack spacing={2}>
+          <Box sx={{ backgroundColor: (theme) => theme.palette.primary.light + '20' }}>
+            <Box padding={2} display="flex">
+              <Typography variant="h2">Edit {dashboardName}</Typography>
+              <Stack direction="row" spacing={1} sx={{ marginLeft: 'auto' }}>
+                <Button variant="contained">Save</Button>
+                <Button variant="outlined" onClick={onCancelButtonClick}>
+                  Cancel
+                </Button>
+              </Stack>
+            </Box>
+          </Box>
+          <Box
+            sx={{
+              display: 'flex',
+              width: '100%',
+              padding: (theme) => theme.spacing(2),
+              transition: 'top 1s cubic-bezier(0.17, 0.04, 0.03, 0.94)',
```

Comment:
Ooo does this just apply as the Box gets rendered

Reply:
no haha i was playing around with animations and forgot to remove it :P